scripts/src/java/org/oppia/android/scripts/todo/IssueTodosCommentCheck.kt

```diff
@@ -3,11 +3,11 @@ package org.oppia.android.scripts.todo
 import java.io.File
 
 /**
- * Script for ensuring that the [ClosedIssueCheck] failure comment is not the same as the latest
- * comment of the closed issue.
+ * Script for ensuring that the [IssueTodosResolvedCheck] failure comment is not the same as the
```

Comment:
The new name actually seems backwards to me. What do you think about 'TodoIssueResolvedCheck' instead? Ditto for the other one & all corresponding doc comments and BUILD targets.

Reply:
Sounds good. Done.